src/containers/shared/components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import copy from '../images/copy.png'
```

Comment:
You should make this an svg.  It will be smaller when gzipped.  Also checkout https://jakearchibald.github.io/svgomg/.